src/common/SloppyCRCMap.cc

```diff
@@ -159,7 +158,7 @@ void SloppyCRCMap::dump(Formatter *f) const
 {
   f->dump_unsigned("block_size", block_size);
   f->open_array_section("crc_map");
-  for (map<uint64_t,uint32_t>::const_iterator p = crc_map.begin(); p != crc_map.end(); ++p) {
+  for (std::map<uint64_t,uint32_t>::const_iterator p = crc_map.begin(); p != crc_map.end(); ++p) {
```

Comment:
if we really need to change this. could use `auto` also. but i'd recommend `using namespace std` to avoid the code churn.